src/Filterable.php

```diff
@@ -11,6 +11,32 @@ trait Filterable
      */
     protected $filtered = [];
 
+    /**
+     * Operator used to filter queries with where.
+     *
+     * @var string
+     */
+    protected $filterWhereOperator;
+
+    /**
+     * Returns the standard operator used for where operations according to the database.
+     */
+    private function getFilterWhereOperator()
+    {
+        if (! isset($this->filterWhereOperator)) {
+            switch (config('database.default')) {
```

Comment:
Instead of getting this from the default config since developers can change this and use different connections, is there a way to get this from the current connection?

Reply:
Hi, thanks for responding.

I did not understand your question, do you say to check more thoroughly as to which db is actually being used without the user specifying this default setting (via string)?

Reply:
Instead of getting the database type from the config, we should get it from the query builder we are currently using in the case that the current query builder instance won't be using the default connection.

Reply:
Ok, I get it. I ended up opting for the configuration of the config / database because I believe that the vast majority of Laravel developers use it (via .env file) and also to use it in the project I'm working on. But we can study and see how it would be possible to implement this, I don't know if it would be something simple, if you have any idea of code it would be welcome.